e2e/cypress/integration/messaging/emoji_size_spec.js

```diff
@@ -16,6 +15,19 @@ function checkEmojiSize(message, emojis) {
     });
 }
 
+/**
+ * [checkEmojiSizeInPost: this function is going to check the correct size of emojis when they're inside messages]
+ * @param  {[type]} message [this is the message we send along with some emojis attached to it ]
+ * @param  {[type]} emojis [array of emojis]
+ */
+function checkEmojiSizeInPost(message, emojis) {
```

Comment:
Could you just combine this with the existing `checkEmojiSize` function?  Something like,
```javascript
function checkEmojiSize(message, emojis, isJumbo) {
    const [height, width, size] = isJumbo ? ['minHeight', 'minWidth', '32px'] : ['height', 'width', '21px'];
    ...
}
```

Reply:
nice idea!!, this is done 👍